src/areaSearch/components/AreaSearchMap.tsx

```diff
@@ -31,11 +32,28 @@ const AreaSearchMap = ({
   blur,
   focus,
 }: WrappedFieldProps & Props) => {
-  const initialPosition = HELSINKI_CENTRAL_COORDINATES;
   const { latLonBounds, CRS } = initializeHelsinkiMap();
 
   const featureGroupRef = useRef<FeatureGroupType>(null);
 
+  const getSelectionAreaCenter = () => {
+    const { coordinates } = value;
```

Comment:
Noticed that this function does not take any arguments. `value` seems like something that should be passed in as an argument

Reply:
Is it a problem that value is provided to the function by closure?

Reply:
Not a problem